components/board/customlinux/data/test_config_empty.json

```diff
@@ -0,0 +1 @@
+{"pins": [{}]}
```

Comment:
These files don't end in newlines. Can they? That's the [POSIX-compliant thing to do](https://thoughtbot.com/blog/no-newline-at-end-of-file), though certain file-parsing utilities are explicitly not POSIX compliant and fail if you have a newline at the end.